.circleci/config.yml

```diff
@@ -815,6 +817,36 @@ jobs:
           paths:
             - desktop\release
 
+  wp-desktop-publish:
```

Comment:
@mokagio This SHA of wp-desktop is a few commits behind that in the main repo, but will ensure to add your script to [validate the version information](https://github.com/Automattic/wp-desktop/pull/939/files#diff-1d37e48f9ceff6d8030570cd36286a61R243) in a downstream PR.

Reply:
Thanks for letting me know 👍